vendor/github.com/v3io/v3io-go-http/container.go

```diff
@@ -123,11 +124,12 @@ func (c *Container) sendRequest(input interface{},
 	// create a request/response (TODO: from pool)
 	requestResponse := &RequestResponse{
 		Request: Request{
-			ID:           id,
-			container:    c,
-			Input:        input,
-			Context:      context,
-			responseChan: responseChan,
+			ID:                  id,
+			container:           c,
+			Input:               input,
+			Context:             context,
+			responseChan:        responseChan,
+			SendTimeNanoseconds: time.Now().UnixNano(),
```

Comment:
Looks like allocation time, not send time.

Reply:
That may be true in your usage of it. It's not true in general. But that's something to take up with upstream I guess.

Reply:
Alright, that's true given the context of how it's created (supposing it's only created here).